extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -171,12 +196,35 @@ export class SchemaCompareResult {
 
 			await view.initializeModel(this.flexModel);
 		});
-	}
 
-	public start(): void {
+		await this.GetDefaultDeploymentOptions();
```

Comment:
nit: if this seems to slow down start we can choose to not wait on it...

Reply:
I haven't noticed any slow down so I'll leave it for now